SpatialGDK/Source/SpatialGDKServices/Public/SSpatialOutputLog.h

```diff
@@ -36,16 +36,16 @@ class SSpatialOutputLog : public SOutputLog
 
 	TUniquePtr<FArchiveLogFileReader> CreateLogFileReader(const TCHAR* InFilename, uint32 Flags, uint32 BufferSize);
 
+	void FormatAndPrintRawLogLine(const FString& LogLine);
+	void FormatAndPrintRawErrorLine(const FString& LogLine);
```

Comment:
I believe we said we would be removing FormatAndPrintRawErrorLine if it is no longer required yah?

Reply:
Actually. A lot of this class can be cleaned up now. Let's do this as part of https://improbableio.atlassian.net/browse/UNR-4336
It's not required for now

Reply:
I thought you said that we are not sure if it is required? Like we are not sure if we have logs formatted this way (and we couldn't get to an error state to test)